packages/permission-controller/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.0.0]
+### Uncategorized
+- Bump Jest to v27 ([#1198](https://github.com/MetaMask/core.git/pull/1198))
+- Restrict certain permissions by subject type ([#1233](https://github.com/MetaMask/core.git/pull/1233))
+- BREAKING: Move `SubjectMetadataController` to permission-controller package ([#1234](https://github.com/MetaMask/core.git/pull/1234))
```

Comment:
Actually this probably isn't breaking for `permission-controller`. It is breaking for `subject-metadata-controller`, which doesn't exist anymore. 🤔